project/make/.ensure-emptyfs

```diff
@@ -1,12 +1,13 @@
 #!/bin/bash
+set -e
 
-if ! docker inspect scratch &> /dev/null; then
+if ! docker inspect emptyfs &> /dev/null; then
 	# let's build a "docker save" tarball for "emptyfs"
 	# see https://github.com/docker/docker/pull/5262
 	# and also https://github.com/docker/docker/issues/4242
-	mkdir -p /docker-scratch
+	dir="$(mktemp -d)"
 	(
-		cd /docker-scratch
+		cd "$dir"
 		echo '{"emptyfs":{"latest":"511136ea3c5a64f264b78b5433614aec563103b4d4702f3ba7d4d2698e22c158"}}' > repositories
```

Comment:
because this is an intentionally empty layer for testing "empty layer" support, not for testing the "scratch as a noop" support (which is why it was renamed to `emptyfs` in the first place)